src/main.rs

```diff
@@ -1,5 +1,3 @@
-#![warn(unused)]
```

Comment:
Why did these get removed?

Reply:
They are on by default now. We get warnings that we are doubling down on overrides